component/alert.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../styles/main.css" />
+    <link rel="stylesheet" href="../styles/alert.css" />
+
+    <title>GK UI</title>
+    <script
+      src="https://kit.fontawesome.com/a4417b6562.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <div class="container">
+      <navbar class="navbar__container box-shadow mb-2">
+        <div class="navbar_nav">
+          <a
+            href="#"
+            class="link__nostyle"
+            style="font-size: 2rem; color: green"
+          >
+            GK UI</a
+          >
+
+          <button class="button">
+            <a href="#" class="link__nostyle">Home</a>
+          </button>
+
+          <button class="button">
+            <a href="#" class="link__nostyle">Installment</a>
+          </button>
+
+          <button class="button">
+            <a href="../pages/documentation.html" class="link__nostyle"
+              >Documnetation
+            </a>
+          </button>
+        </div>
+
+        <div class="navbar__git-icon">
+          <button class="button btn-icon">
+            <a
+              href="https://github.com/shubhamkumar648"
+              class="link__nostyle"
+              target="_blank"
+              ><i class="fab fa-github"></i
+            ></a>
+          </button>
+        </div>
+      </navbar>
+
+      <aside class="aside__container p-2">
+        <div class="aside__header">
+          <span class="fs-xl">Component</span>
+        </div>
+
+        <hr />
+
+        <div class="aside__content p-1">
+          <span>
+            <a class="active link__nostyle" href="../component/avatar.html"
+              >Avatar</a
+            ></span
+          >
+
+          <span>
+            <a class="active link__nostyle" href="../component/alert.html"
+              >Alert</a
+            ></span
+          >
+          <span> <a class="active link__nostyle" href="#">Home</a></span>
+          <span> <a class="active link__nostyle" href="#home">Home</a></span>
+
+          <span> <a class="active link__nostyle" href="#home">Home</a></span>
+          <span> <a class="active link__nostyle" href="#home">Home</a></span>
+        </div>
+      </aside>
+
+      <main class="main__container p-2">
+        <h3 class="mb-1">Alert</h3>
+        <h6 class="mb-2">Example of Alert</h6>
+        <p class="mb-1">
+          It can be used to notify the user. Variant:
+          danger||success||information|| warning.
+        </p>
+        <div class="alert__container flex-col p-2 mb-2">
+          <div class="alert info">
```

Comment:
You can give the class name `alert-info` instead of `info`.
This will avoid conflicts in future projects.
(Apply to all)

Reply:
thank you done